### PR TITLE
check that args is a string before using - else raycast may pass empt…

### PR DESCRIPTION
**Pull request by GastroGeek**
…y arguments object

## Description

```
git commit -m "check that args is a string before using - else raycast may pass empty arguments object"
```

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by mooxl** [comment]
LGTM and fixes #2329

**Comment by GastroGeek** [comment]
> 

forgive my potential ignorance but do you mean in the Caffeine() function? something like

```
const Caffeinate = async (args: "") => 
```

? only that won't help I don't think since the problem is when you execute just `Caffeinate` Raycast sends `{ arguments: {} }` which overrides that default... so same problem?

File: extensions/coffee/src/caffeinate.ts [hunk 1]
```diff
@@ -2,11 +2,13 @@ import { showHUD } from "@raycast/api";
...
 
 const Caffeinate = async (args?: string) => {
+  const validStringCaffeinateArgs = typeof args === "string";
+
   try {
     await runAppleScript(`do shell script "pgrep caffeinate"`);
     await showHUD("Your Mac is already caffeinated");
   } catch (_) {
-    runAppleScript(`do shell script "caffeinate -di${args ? ` ${args}` : ""}"`);
+    runAppleScript(`do shell script "caffeinate -di${validStringCaffeinateArgs ? ` ${args}` : ""}"`);
```
Comment (mooxl) [inline extensions/coffee/src/caffeinate.ts:11]: just a nit: `${typeof args === "string" ? ...}` would save one line:)

**Review by mooxl — APPROVED** [review]

File: extensions/coffee/src/caffeinate.ts (hunk 1 above)
Comment (GastroGeek) [reply extensions/coffee/src/caffeinate.ts]: should be on PR now 😅 that's the hardest _simple_ fix, I've ever done heh.

Comment (mooxl) [reply extensions/coffee/src/caffeinate.ts]: if you want, you could also add yourself to the contributors in the package.json with that one liner :) @GastroGeek

Comment (GastroGeek) [reply extensions/coffee/src/caffeinate.ts]: why not, eh? 😄 done! thanks!

**Review by mattisssa — APPROVED** [review]
Let's get it merge until we introduce the fix in the next release. 🚀 
Sorry for inconvenience

**Pull request merged**